operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/Utils.java

```diff
@@ -48,4 +49,13 @@ public static Version loadFromProperties() {
         properties.getProperty("git.commit.id.abbrev", "unknown"),
         builtTime);
   }
+
+  public static boolean isValidateCustomResourcesEnvVarSet() {
```

Comment:
again.. customResourceDefinition instead of customResource. I know it's long but it's confusing otherwise